bundle/tests/scorecard/kuttl/semeru/01-assert.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 60
+---
+apiVersion: apps/v1
```

Comment:
These asserts could be part of 00-assert.yaml since JIT server should be available prior to application deployment.